config.default.yaml

```diff
@@ -476,15 +477,16 @@ export:
 
 custom_data:
   renewables: [] # ['csp', 'rooftop-solar', 'solar']
-  elec_demand: false
-  heat_demand: false
+  sectoral_energy_demand: false
+  # elec_demand: false
+  # heat_demand: false
   industry_demand: false
   industry_database: false
-  transport_demand: false
+  # transport_demand: false
   water_costs: false
   h2_underground: false
   add_existing: false
-  custom_sectors: false
+  # custom_sectors: false
```

Comment:
From the previous experience, commented-out lines tend to stay in the code forever. The reason of commenting-out gets forgotten quite quickly, and you can't be sure for which particular purpose the commented-out stuff is needed and if it can be removed safely.

Would you mind to remove the unneeded parameters instead commenting-out them?